wallet/src/main/java/io/neow3j/wallet/Nep5.java

```diff
@@ -0,0 +1,75 @@
+package io.neow3j.wallet;
+
+import io.neow3j.contract.ContractParameter;
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.methods.response.NeoInvoke;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+
+public class Nep5 {
+
+    private Neow3j neow3j;
+    private ScriptHash fromContractScriptHash;
+
+    public Nep5() {
+    }
+
+    private Nep5(Builder builder) {
+        this.neow3j = builder.neow3j;
+        this.fromContractScriptHash = builder.fromContractScriptHash;
+    }
+
+    public BigInteger totalSupply() throws IOException {
+        List<ContractParameter> params = Arrays.asList(
+                ContractParameter.string("totalSupply"),
+                ContractParameter.array());
+        NeoInvoke send = neow3j.invoke(this.fromContractScriptHash.toString(), params).send();
+        return send.getInvocationResult().getStack().get(1).asByteArray().getAsNumber();
+    }
+
+    public String name() throws IOException {
+        List<ContractParameter> params = Arrays.asList(
+                ContractParameter.string("name"),
+                ContractParameter.array());
+        NeoInvoke send = neow3j.invoke(this.fromContractScriptHash.toString(), params).send();
+        return send.getInvocationResult().getStack().get(1).asByteArray().getAsString();
+    }
+
+    public String symbol() throws IOException {
+        List<ContractParameter> params = Arrays.asList(
+                ContractParameter.string("symbol"),
+                ContractParameter.array());
+        NeoInvoke send = neow3j.invoke(this.fromContractScriptHash.toString(), params).send();
+        return send.getInvocationResult().getStack().get(1).asByteArray().getAsString();
+    }
+
+    public BigInteger decimals() throws IOException {
+        List<ContractParameter> params = Arrays.asList(
+                ContractParameter.string("decimals"),
+                ContractParameter.array());
+        NeoInvoke send = neow3j.invoke(this.fromContractScriptHash.toString(), params).send();
+        return (BigInteger) send.getInvocationResult().getStack().get(1).getValue();
```

Comment:
At the moment we have to use `send.getInvocationResult().getStack().get(1)`.
The only thing we can change here is use `asInteger().getValue()` instead of just `getValue()` and then casting it to BigInteger.